.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -39,6 +39,13 @@ const getPipeline = (filename: string, removeSteps = true) => {
       return;
     }
 
+    const onlyRunQuickChecks = await areChangesSkippable([/^renovate\.json$/], REQUIRED_PATHS);
+    if (onlyRunQuickChecks) {
+      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/renovate.yml'));
```

Comment:
Do I need to pass in `true` or `false` for the `removeSteps` parameter?

Reply:
`false`